independent-publisher-connectors/Working days/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2104 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.2.0",
+    "title": "Working days",
+    "description": "Working days API. It covers more than 45 countries and 230 regional calendars. It also covers main stock exchanges calendars. The quality of data is the top priority. API offers up to date information by following government announcements regarding public holidays changes.",
```

Comment:
@tposzytek word `API` is not allowed in the description, please remove it. The description itself should reflect the main value and purpose offered by the connector. More information about the description can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector).